vscode-trace-extension/src/trace-explorer/opened-traces/trace-explorer-opened-traces-view-provider.ts

```diff
@@ -0,0 +1,128 @@
+import * as vscode from 'vscode';
```

Comment:
I had to scroll back and forth a few time to catch the difference with the below `trace-explorer-views-provider.ts` file. Should that other file be `trace-explorer-views-view-provider.ts`? or replace the `view` in the name by `webview` and do the same below?

Reply:
yeah, I struggle myself keeping track with the different files. I think adding "webview" makes totally sense and helps with clarity.